client/components/marketing-survey/cancel-jetpack-form/jetpack-cancellation-offer.tsx

```diff
@@ -0,0 +1,202 @@
+					'Your biennial subscription renews every two years. It will renew at {{strong}}%(renewalPrice)s/biennium{{/strong}} for the next %(periods)d bienniums. It will then renew at {{strong}}%(fullPrice)s/biennium{{/strong}} each following biennium.',
+					renewalCopyOptions
+				);
+				if ( 1 === periods ) {
+					/* Translators: %(discount)d%% is a discount percentage like 15% or 20% */
+					offerHeadline = translate(
+						'Get %(discount)d%% off %(name)s for your next two-year renewal',
+						headlineOptions
+					);
+					renewalCopy = translate(
+						'Your biennial subscription renews every two years. It will renew at {{strong}}%(renewalPrice)s/biennium{{/strong}} for the next biennium. It will then renew at {{strong}}%(fullPrice)s/biennium{{/strong}} each following biennium.',
+						renewalCopyOptions
+					);
+				}
+				break;
+			case PLAN_ANNUAL_PERIOD:
+				/* Translators: %(discount)d%% is a discount percentage like 15% or 20% */
+				offerHeadline = translate(
+					'Get %(discount)d%% off %(name)s for the next %(periods)d years',
+					headlineOptions
+				);
+				renewalCopy = translate(
+					'Your annual subscription will renew at {{strong}}%(renewalPrice)s/year{{/strong}} for the next %(periods)d years. It will then renew at {{strong}}%(fullPrice)s/year{{/strong}} each following year.',
+					renewalCopyOptions
+				);
+				if ( 1 === periods ) {
+					/* Translators: %(discount)d%% is a discount percentage like 15% or 20% */
+					offerHeadline = translate(
+						'Get %(discount)d%% off %(name)s for the next year',
+						headlineOptions
+					);
+					renewalCopy = translate(
+						'Your annual subscription will renew at {{strong}}%(renewalPrice)s/year{{/strong}} for the next year. It will then renew at {{strong}}%(fullPrice)s/year{{/strong}} each following year.',
+						renewalCopyOptions
+					);
+				}
+				break;
+			case PLAN_MONTHLY_PERIOD:
+				/* Translators: %(discount)d%% is a discount percentage like 15% or 20% */
+				offerHeadline = translate(
+					'Get %(discount)d%% off %(name)s for the next %(periods)d months',
+					headlineOptions
+				);
+				renewalCopy = translate(
+					'Your monthly subscription will renew at {{strong}}%(renewalPrice)s/month{{/strong}} for the next %(periods)d months. It will then renew at {{strong}}%(fullPrice)s/month{{/strong}} each following month.',
+					renewalCopyOptions
+				);
+				if ( 1 === periods ) {
+					/* Translators: %(discount)d%% is a discount percentage like 15% or 20% */
+					offerHeadline = translate(
+						'Get %(discount)d%% off %(name)s for the next year',
+						headlineOptions
+					);
+					renewalCopy = translate(
+						'Your monthly subscription will renew at {{strong}}%(renewalPrice)s/month{{/strong}} for the next month. It will then renew at {{strong}}%(fullPrice)s/month{{/strong}} each following month.',
+						renewalCopyOptions
+					);
+				}
+		}
+
+		return { offerHeadline, renewalCopy };
+	}, [ offer, percentDiscount, purchase ] );
```

Comment:
`translate` should be part of the deps too.

Reply:
indeed, thanks!